<launch>
  <arg name="robosherlock_sim" default="False" />
  <arg name="ceiling_height" default="2.2" />
  <arg name="json_file" default="" />
  <arg name="initial_beliefstate_owl" default="" />
  <arg name="rgb_topic" default="/refills_wrist_camera/image_color" />
  <arg name="realsense_topic" default="/rs_camera/color/camera_info" />


  <node name="perception_interface" pkg="refills_perception_interface" type="perception_interface.py" output="screen">
    <param name="initial_beliefstate" value="$(arg initial_beliefstate_owl)" />
    <param name="robosherlock_sim" value="$(arg robosherlock_sim)" />
    <param name="ceiling_height" value="$(arg ceiling_height)" />
    <param name="path_to_json" value="$(arg json_file)" />
    <param name="rgb_topic" value="$(arg rgb_topic)" />
    <param name="realsense_topic" value="$(arg realsense_topic)" />
    <remap from="/separator_marker_detector_node/data_out" to="/separator_marker_detector_node/data_out"/>
    <remap from="/barcode/pose" to="/barcode/pose"/>
  </node>

</launch>
